lib/assets.js

```diff
@@ -12,14 +12,14 @@ var defaults = {
   }
 };
 
-module.exports = function (gulp, options) {
-  var opts = merge({}, defaults, options);
-  var dest = opts.dest;
-  var optimize = opts.optimize;
-  var plugins = opts.plugins;
-  var src = opts.src;
+module.exports = (gulp, options) => {
+  const opts = merge({}, defaults, options);
```

Comment:
@mrgerardorodriguez 

> I was under the impression that my options were either let or const for ES2015/ES6. I didn't realize I could still use var.

`var` is still the go-to. Many folks (myself included) use `let` when it's not really necessary. You only _need_ `let` when you need block-scoping (and I'd say even the need for that is uncommon).

Examples of `let` being used for defining block-scoped variables:

```js
if (something) {
  let somethingIsTrue = true
}
```

```js
for (let index = 0; index < items.length; index++) {
  // ...
}
```

Here's one article that explains the usage well: https://davidwalsh.name/for-and-against-let